lib/thirdparty/LBRY.class.php

```diff
@@ -75,4 +75,9 @@ public static function editYouTube($status_token, $channel_name, $email, $sync_c
             return Curl::post(static::getApiUrl("/yt/update"), ['status_token' => $status_token, 'new_email' => $email, 'new_preferred_channel' => $channel_name, 'sync_consent' => $sync_consent], ['json_response' => true]);
         }
     }
+
+    public static function logWebVisitor($site, $visitorID, $IPAddress)
+    {
+        return Curl::post(static::getApiUrl("/visitor/new"), ['site' => $site, 'visitor_id' => $visitorID, 'ip_address' => $IPAddress], ['json_response' => true]);
```

Comment:
Consider how to handle this function only being called in production.

Reply:
I can add the `IS_PRODUCTION` check here and only return the curl post if that is true. I tested this out and it does work and stops the api from being called. What I am not sure of are the implications. If I return nothing, what does that mean in PHP? There are no errors in the console and the web page appears fine.

Reply:
This function probably doesn't need to return at all, since the return is never checked or used anyway.

Reply:
well it is used to store the visitor id in the php session object. This is how we give an id to the analytics.

Reply:
Right, my bad.